cli/src/actions/template.ts

```diff
@@ -40,14 +40,41 @@ export async function promptTemplate(ctx: Context) {
 }
 
 export async function promptOpenAPI(ctx: Context) {
-  const confirmOpenAPI = await confirm({
-    message: "Do you need an OpenAPI spec?",
-    initialValue: true,
-    active: "Yes",
-  });
-
-  if (typeof confirmOpenAPI === "boolean" && confirmOpenAPI) {
-    ctx.useOpenAPI = confirmOpenAPI;
+  try {
+    const confirmOpenAPI = await confirm({
+      message: "Do you need an OpenAPI spec?",
+      initialValue: true,
+      active: "Yes",
+    });
+
+    if (typeof confirmOpenAPI === "boolean" && confirmOpenAPI) {
+      const result = await select({
+        message: "Which openapi lib do you want to use?",
+        options: [
+          {
+            value: "openapi",
+            label: "Zod OpenAPI template",
+            hint: "This uses @hono/zod-openapi library",
```

Comment:
can you add just a little bit more context for both values? Might be worth mentioning that the Hono OpenAPI library is easier to adopt as it runs as middleware and extends from the "vanilla" Hono class while Zod OpenAPI offers better type checking but is a different Hono subclass.